pkg/clusterapi/aws/actuator.go

```diff
@@ -542,3 +529,28 @@ func getBootstrapKubeconfig() (string, error) {
 func iamRole(cluster *clusterv1.Cluster) string {
 	return fmt.Sprintf("%s", defaultIAMRole)
 }
+
+func buildDescribeSecurityGroupsInput(vpcID, vpcName string, isMaster, isInfra bool) *ec2.DescribeSecurityGroupsInput {
+	groupNames := []*string{aws.String(vpcName)}
+	if isMaster {
+		groupNames = append(groupNames, aws.String(vpcName+"_master"))
+		groupNames = append(groupNames, aws.String(vpcName+"_master_k8s"))
+	}
+	if isInfra {
+		groupNames = append(groupNames, aws.String(vpcName+"_infra"))
+		groupNames = append(groupNames, aws.String(vpcName+"_infra_k8s"))
+	}
+	if !isMaster && !isInfra {
+		groupNames = append(groupNames, aws.String(vpcName+"_compute"))
+		groupNames = append(groupNames, aws.String(vpcName+"_compute_k8s"))
+	}
+
+	sgNameFilter := "group-name"
+	describeSecurityGroupsInput := ec2.DescribeSecurityGroupsInput{
+		Filters: []*ec2.Filter{
+			{Name: aws.String("vpc-id"), Values: []*string{&vpcID}},
+			{Name: &sgNameFilter, Values: groupNames},
```

Comment:
For consistency with the rest of the function, use `aws.String("group-name")` here.